main.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // This will be loaded from env later I'm assuming. For now just returns what port is used on localhost.
-const HOST = "127.0.0.1"
+const HOST = "0.0.0.0"
```

Comment:
why this change?

Reply:
Was going to ask you about this. To externally test if the server is running properly I have to use 0.0.0.0 to access the container via it's IP and the port. Alternatively I can exec into the container, install curl and confirm that it's working by curling localhost:8080.

What I'm having a hard time understanding is which we are looking for at this point? If we're going to direct traffic to it eventually wouldn't it need to be 0.0.0.0? Or does it not matter since a K8 service will handle that connection later?

Reply:
Good, yeah so when you bind to `127.0.0.1` that means you'll reject anything that does not come from the same host (more or less, don't worry about it) and if you bind to `0.0.0.0` it'll accept from anywhere.  and if you set it to `244.10.8.24` it'll only accept connections from that specific host.

So we want it to default to 127.0.0.1 so that if we mistakenly forget to set it it'll be secure.  But now that we ran into the issue of needing it to be multiple values, it's time to make this configurable at runtime.  Either via environment variables or a config file (or both) we need to be able to set this when we run it. 

- https://github.com/spf13/cobra
- https://github.com/spf13/viper